web/src/containers/Routes/index.tsx

```diff
@@ -11,10 +11,28 @@ import { withScrollRestoration } from "../../utility/withScrollRestoration";
 import { ReactNode } from "react";
 import { useRecoilValue } from "recoil";
 
+const showGemsOnly = false;
+
+function FilterGemsOnly(taskItems: TaskListItem[]) {
```

Comment:
non-component functions are generally camel-cased (filterGemsOnly)

Reply:
No clue what components are you talking about but ok

Reply:
react functional components, RoutesContainer below is one of them (you can tell because it returns JSX, which is the HTML-like code)